### PR TITLE
Updates for NP after final code review and fix customer name formatting

### PR DESCRIPTION
**Pull request by mateuszgrzyb**
I want to merge this change because...

SALEOR-4969

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4969 Comments from final PR](https://app.clickup.com/t/2549495/SALEOR-4969)

**Comment by L3str4nge** [comment]
Why this PR has 125 commits?

**Comment by mateuszgrzyb** [comment]
> Why this PR has 125 commits?

I've created this branch from NP feature branch instead of `3.0` and if I understand correctly, previous NP pull request was merged with squash. Should I just create a new pr with all of this commits squashed?

**Comment by patrys** [comment]
@mateuszgrzyb Can you rebase it on the current branch so it only contains the relevant commits?

**Review by L3str4nge — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

File: saleor/payment/gateways/np_atobarai/__init__.py [hunk 1]
```diff
@@ -18,28 +21,22 @@
...
-        return fun(payment_information, get_api_config(config.connection_params))
-
-    return inner
-
-
 def parse_errors(errors: List[str]) -> str:
-    # FIXME: better solution?
-    #  Transaction.error in database has max_length of 256
-    return os.linesep.join(errors)[:256]
+    return os.linesep.join(errors[:11])
```
Comment (stnatic) [inline saleor/payment/gateways/np_atobarai/__init__.py:25]: Could we add a comment here? Previously I could understand the slice and the db constraint and accept the fact that 256 characters will be enough to debug stuff. In the current implementation 11 looks like a magic number

File: saleor/payment/gateways/np_atobarai/__init__.py [hunk 2]
```diff
@@ -18,28 +21,26 @@
...
     payment_information: PaymentData, config: ApiConfig
 ) -> GatewayResponse:
     payment_id = payment_information.payment_id
     payment = Payment.objects.filter(pk=payment_id).first()
 
     if not payment:
-        raise PaymentError(f"Payment with id {payment_id} does not exist.")
+        logger.error(
+            "Payment with id %s does not exist",
+            Node.to_global_id("Payment", payment_id),
```
Comment (korycins) [inline saleor/payment/gateways/np_atobarai/__init__.py:41]: You can use here `payment_information.graphql_payment_id`

File: saleor/payment/gateways/np_atobarai/api.py [hunk 3]
```diff
@@ -144,6 +159,10 @@ def change_transaction(
             )
 
         if PRE_FULFILLMENT_ERROR_CODE in error_codes:
+            logger.info(
+                "Fulfillment for payment with id %s was reported",
+                Node.to_global_id("Payment", payment.id),
```
Comment (korycins) [inline saleor/payment/gateways/np_atobarai/api.py:164]: `payment_information.graphql_payment_id`

File: saleor/payment/gateways/np_atobarai/tests/test_api.py [hunk 4]
```diff
@@ -5,6 +5,8 @@
 import pytest
 import requests
 
+from saleor.order import OrderEvents
```
Comment (korycins) [inline saleor/payment/gateways/np_atobarai/tests/test_api.py:8]: You are using relative imports in the file, except this one.

File: saleor/payment/gateways/np_atobarai/tests/test_plugin.py [hunk 5]
```diff
@@ -1,7 +1,12 @@
+import logging
 import os
-from unittest.mock import Mock, patch
+from unittest.mock import ANY, Mock, patch
 
+import pytest
 import requests
+from graphene import Node
+
+from saleor.payment import PaymentError
```
Comment (korycins) [inline saleor/payment/gateways/np_atobarai/tests/test_plugin.py:9]: Would be good to have unified way of imports between files. Looks like in the rest of the test files you're using relative.

File: saleor/payment/gateways/np_atobarai/tests/test_api.py (hunk 4 above)
Comment (fowczarek) [inline saleor/payment/gateways/np_atobarai/tests/test_api.py:8]: @mateuszgrzyb  Absolute import, we should use relative import.

File: saleor/payment/gateways/np_atobarai/tests/test_plugin.py (hunk 5 above)
Comment (fowczarek) [inline saleor/payment/gateways/np_atobarai/tests/test_plugin.py:9]: @mateuszgrzyb Absolute import, we should use relative import.

**Pull request merged**